pkgs/applications/audio/clementine/default.nix

```diff
@@ -104,46 +106,26 @@ let
       mkdir -p $out/libexec/clementine
       mv $out/bin/clementine-spotifyblob $out/libexec/clementine
       rmdir $out/bin
+
+      makeWrapper ${free}/bin/clementine $out/bin/clementine \
+        --set CLEMENTINE_SPOTIFYBLOB $out/libexec/clementine \
+        --prefix GST_PLUGIN_SYSTEM_PATH_1_0 : "$GST_PLUGIN_SYSTEM_PATH_1_0"
+
+      mkdir -p $out/share
+      for dir in applications icons kde4; do
+        ln -s "$free/share/$dir" "$out/share/$dir"
+      done
     '';
     enableParallelBuilding = true;
     meta = with stdenv.lib; {
       homepage = http://www.clementine-player.org;
       description = "Spotify integration for Clementine";
       # The blob itself is Apache-licensed, although libspotify is unfree.
-      license = licenses.asl20;
+      # Therefore the license will be marked as `unfree` because of its dependencies.
+      license = licenses.unfree;
```

Comment:
Not sure whether that is needed. Because of its dependency on `libspotify` the leaf will be unfree and thus this package would fail to build or install because it requires unfree software.

Reply:
I'll change that for now. The original issue was filed because of the package shipped with a free license, but `libspotify` broke the evaluation on systems which disallow unfree packages. However the affected package is now named `clementineUnfree`, so I guess that it should be sufficiently obvious :)